picturae_importer.py

```diff
@@ -38,29 +38,58 @@ def __init__(self, paths, config, date_string=None):
 
         self.logger = logging.getLogger("PicturaeImporter")
 
-        self.init_all_vars(date_string=date_string, paths=paths)
+        self.csv_folder = self.picturae_config.CSV_FOLDER
 
-        self.record_full = pd.read_csv(self.file_path)
+        self.process_csv_files(date=date_string)
+
+        self.init_all_vars()
 
         self.num_barcodes = len(self.record_full)
 
         self.run_all_methods()
 
```

Comment:
new breakoff function from init all vars, commited to reading in and re-processing image paths from csv files.